firmware/tunerstudio/rusefi.input

```diff
@@ -967,13 +967,13 @@ menuDialog = main
 		subMenu = std_separator
 		subMenu = cltIdleCurve,				"Warmup Idle multiplier"
 
-	menu = "Tuning"
```

Comment:
Rearrange menu to group all 'Enrichment' items together:
![image](https://user-images.githubusercontent.com/15104768/37730155-967e3a3e-2d47-11e8-955e-d527ef2fe792.png)